cli/setup.py

```diff
@@ -73,12 +73,15 @@ def version(default):
       url='https://github.com/hyperledger/sawtooth-core',
       packages=find_packages(),
       install_requires=[
-          'sawtooth-signing',
           'colorlog',
-          'protobuf'
+          'grpcio-tools',
```

Comment:
`grpcio-tools` is the package that was being pip3 installed, so that is the package I built a deb for and installed.

I can test it locally to see if it still works with just protobuf...